instructor_notes/04_shell-scripting.md

```diff
@@ -13,16 +13,26 @@ At the end of this activity, workshop attendees should be able to:
 + Write a simple shell script with useful skills like defining, writing a for loop, and printing
 
 
-## Activity 
+## Activity
 
 The goal of this activity is to interactively develop a script to download and save these two paired FASTQ files:
 
 + Paired FASTQ files from (Study ID SRP255885)[https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP255885], and RNA-seq dataset about medulloblastoma
-  + Download links: 
+  + Download links:
     + https://sra-download.ncbi.nlm.nih.gov/traces/sra63/SRZ/011518/SRR11518889/NC16_S1_L004_R1_001.fastq.gz
     + https://sra-download.ncbi.nlm.nih.gov/traces/sra63/SRZ/011518/SRR11518889/NC16_S1_L004_R2_001.fastq.gz
   + Compressed files are 105.9 and 116.5 MB, which is not too big to slam their computers but also too big for github
 
++ **Alternatively**, if internet speed is prohibitively slow for downloading these full FASTQ files, we have subsetted versions that contain only the first 10,000 reads.
+  + Download links:
+    + https://raw.githubusercontent.com/AlexsLemonade/reproducible-research/main/instructor_notes/fastq_subset/NC16_S1_L004_R1_001.fastq.gz
```

Comment:
The only reason I can think to use S3 here would be for shorter links. We probably can't use a link shortener though, unless we find one that lets us keep the file name at the end.